examples/src/providers/rule_closures/conditions.js

```diff
@@ -30,6 +30,32 @@ module.exports = [
       }
     },
   },
+  {
+    /**
+     * invalidOrderUdp
+     * example {closure: "invalidOrderUdp"} - If the order even valid
```

Comment:
Grammar here. Maybe should say
```suggestion
     * example {closure: "invalidOrderUdp"} - Is the order even valid?
```